added a value attribute to the attack pattern ioc (#19034)
demisto/content: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_1.md

#### Integrations

> - **MITRE ATT&CK Feed v2**
> - Improved the human-readable in the ***attack_pattern_reputation_command***.
> - Added a *AttackPattern.Value* to the context entry of the ***attack_pattern_reputation_command***.

Reviewer: ```suggestion
##### MITRE ATT&CK Feed v2
- Improved the human-readable in the ***attack-pattern*** command.
- Added the *AttackPattern.Value* output to the ***attack-pattern*** command.
```
